bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VLT_ta/VLT_ta_M1600/VLT_ta_M1600_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+
+define l+ = l+ ta+
+define l- = l- ta-
+import model NewResFlags
+generate p p > uq4 > t a, ( t > b w+ , w+ > l+ vl) @0
+add process  p p > uq4~ > t~ a, ( t~ > b~ w- , w- > l- vl~) @1
+add process  p p > uq4 > t a, ( t > b w+ , w+ > j j) @2
+add process  p p > uq4~ > t~ a, ( t~ > b~ w- , w- > j j) @3
```

Comment:
are these used also for UL, do you need the spin correlation in the final, if not decaying top in pythia can significantly reduce the number of file and production speed

Reply:
Yes, these are used for Run2_UL as well. Generally, we don't need the spin correlation, but if we want to consider spin correlation, should I use MadSpin? Regarding top decay, please keep top decay as it is.

Reply:
the decay chain also keeps the spin correlation.

Reply:
The decay chain as mentioned above?
I just realized that in the run card, I forgot to change the beam energy to 6800 GeV, so I need to modify the run card.
If there aren't any other modifications, can I upload the new run card to proceed to the next step?

Reply:
yes, please

Reply:
Hi @menglu21,
I have uploaded the modified run card. Please check it and let me know if everything works well.